README.md

```diff
@@ -8,8 +8,8 @@ Java and Android library to translate OSM tags to and from localized names.
 
 ## Possible use cases
 
-* Have a user find features on the map by searching for their common names, synonyms or keywords for that primary feature. For example, if the user inputs "supermarket", he wants to find all supermarkets - and not all places with "supermarket" in its name.
+* Have a user find features on the map by searching for their common names, synonyms or keywords for that primary feature. For example, if the user inputs "supermarket", he/she wants to find all supermarkets - and not all places with "supermarket" in its name.
 
 * Correctly refer to a feature with a set of tags from the map (of which detailed information is shown). For example  the "Query features" feature on openstreetmap.org.
 
-* When editing the map, a quick way to find the correct tag(s) for a given term
\ No newline at end of file
+* When editing the map, a quick way to find the correct tag(s) for a given term
```

Comment:
This was changed by GitHub online editor, likely just line ending at the end of the file…